README.md

```diff
@@ -14,7 +14,7 @@ GraalVM to compile Java code to a native library.
 
 ## Documentation
 
-Latest version of the documentation with API reference and many code samples is [here](https://readthedocs.org/projects/pypowsybl/badge/?version=latest).  
+Latest version of the documentation with API reference and many code samples is [here](https://pypowsybl.readthedocs.io/en/latest/?badge=latest).  
```

Comment:
Thanks for identifying this issue : 
actually I think we should instead link to the last stable version of the doc : 
https://pypowsybl.readthedocs.io/en/stable/